src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -109,8 +109,8 @@ const BioAndDMSetting = ({
           value={isAnonymity}
           onValueChange={toggleSwitchAnon}
           labelLeft="Allow anonymous messages?"
-          circleActiveColor={colors.blue1}
-          activeTextColor={colors.blue1}
+          circleActiveColor={COLORS.blue}
+          activeTextColor={COLORS.blue}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated to use the new `COLORS` constant. This change ensures consistency across the application and makes it easier to manage color schemes.
```diff
- circleActiveColor={colors.blue1}
- activeTextColor={colors.blue1}
+ circleActiveColor={COLORS.blue}
+ activeTextColor={COLORS.blue}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->